### ISSUE TITLE
support for OpenAI models

### ISSUE THREAD

**Issue by mmg10** [mkurman/mcts-pytorch#1]
Hi,

Is there a way to make this work with OpenAI (or any other API based) models?

**Comment by mkurman** [comment]
It is not possible until you have access to the logits. Access to logits is essential for calculating the loss, as we use a scoring function that requires float values. 

You can view the method used to calculate the score here: https://github.com/mkurman/mcts-pytorch/blob/71f280a859f80a2b9647d9f4eef5e7c3bec90caa/src/mcts.py#L183
